test/bc_coeff_compositions.jl

```diff
@@ -197,19 +197,19 @@ end
 @testset "Test Left Division L4 (fourth order)" begin
 
     # Test \ homogenous and inhomogenous BC
-    dx = 0.00001
-    x = 0.0001:dx:0.01
+    dx = 0.01
+    x = 0.0001:dx:0.2

```

Comment:
why is the left end so many zeros if dx = 0.01?

Reply:
I'll fix that, I forgot to change the left end.